pilot/pkg/networking/util/util.go

```diff
@@ -511,7 +511,7 @@ func BuildLbEndpointMetadata(network, tlsMode, workloadname, namespace, clusterI
 		addIstioEndpointLabel(metadata, "network", &pstruct.Value{Kind: &pstruct.Value_StringValue{StringValue: network}})
 	}
 
-	if tlsMode != "" && tlsMode != model.DisabledTLSModeLabel {
```

Comment:
@bianpengyuan I see you added this conditional. We rely on the `tlsMode` metadata elsewhere in EDS. 

Is including `tlsMode: disabled` in the metadata problematic? I can work around that if so.